adapters/automatad/automatad_test.go

```diff
@@ -0,0 +1,20 @@
+package automatad
+
+import (
+	"testing"
+
+	"github.com/prebid/prebid-server/adapters/adapterstest"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+func TestJsonSamples(t *testing.T) {
+	bidder, buildErr := Builder(openrtb_ext.BidderAutomatad, config.Adapter{
+		Endpoint: "http://whatever.url"})
```

Comment:
Could you please rename it to something like "biddertest.com"?

Reply:
I've updated it to "biddertest.com" across the tests.